edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/identity/service/ServiceIdentity.cs

```diff
@@ -48,9 +52,19 @@ public class ServiceIdentity : IEquatable<ServiceIdentity>
         [JsonConverter(typeof(OptionConverter<string>))]
         public Option<string> ModuleId { get; }
 
+        [JsonProperty("deviceScope")]
+        [JsonConverter(typeof(OptionConverter<string>))]
+        public Option<string> DeviceScope { get; }
+
+        [JsonProperty("parentScopes")]
+        public ISet<string> ParentScopes { get; }
```

Comment:
IotHub returns a IList, it should be in theory a set, but just in case that changes should use a list here too